docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.md

```diff
@@ -0,0 +1,422 @@
+> kubectl port-forward svc/camunda-tasklist 8082:80
+Optimize:
+> kubectl port-forward svc/camunda-optimize 8083:80
+Connectors:
+> kubectl port-forward svc/camunda-connectors 8088:8080
+```
+
+Please keep in mind that KeyCloak has to be port-forwarded at all times as it is required to authenticate.
+
+```shell
+kubectl port-forward services/camunda-keycloak 18080:80
+```
+
+  </TabItem>
+    <TabItem value="modeler" label="Modeler">
+
+Follow our existing [Modeler guide on deploying a diagram](../../../../../modeler/desktop-modeler/deploy-to-self-managed/). Below are the helper values required to be filled in Modeler.
+
+<Tabs groupId="domain">
+  <TabItem value="with" label="With Domain">
+
+The following values are required for the OAuth authentication:
+
+```shell
+# Make sure to manually replace #DOMAIN_NAME with your actual domain since the Modeler can't access the shell context
+Cluster endpoint=https://zeebe.$DOMAIN_NAME
+Client ID='client-id' # retrieve the value from the identity page of your created m2m application
+Client Secret='client-secret' # retrieve the value from the identity page of your created m2m application
+OAuth Token URL=https://$DOMAIN_NAME/auth/realms/camunda-platform/protocol/openid-connect/token
+Audience=zeebe-api # the default for Camunda 8 self-managed
+```
+
+  </TabItem>
+  <TabItem value="without" label="Without Domain">
+
+This requires to port-forward the Zeebe Gateway and Keycloak to be able to connect to the cluster.
+
+```shell
+kubectl port-forward services/camunda-zeebe-gateway 26500:26500
+kubectl port-forward services/camunda-keycloak 18080:80
+```
+
+The following values are required for the OAuth authentication:
+
+```shell
+# Make sure to manually replace #DOMAIN_NAME with your actual domain since the Modeler can't access the shell context
+Cluster endpoint=http://localhost:26500
+Client ID='client-id' # retrieve the value from the identity page of your created m2m application
+Client Secret='client-secret' # retrieve the value from the identity page of your created m2m application
+OAuth Token URL=http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token
+Audience=zeebe-api # the default for Camunda 8 self-managed
+```
+
+If you want to access the other services and their UI, you can port-forward those as well:
+
+```shell
+Identity:
+> kubectl port-forward svc/camunda-identity 8080:80
+Operate:
+> kubectl port-forward svc/camunda-operate  8081:80
+Tasklist:
+> kubectl port-forward svc/camunda-tasklist 8082:80
+Optimize:
+> kubectl port-forward svc/camunda-optimize 8083:80
+Connectors:
+> kubectl port-forward svc/camunda-connectors 8088:8080
+```
+
+Please keep in mind that KeyCloak has to be port-forwarded at all times as it is required to authenticate.
+
+```shell
+kubectl port-forward services/camunda-keycloak 18080:80
+```
+
+  </TabItem>
+</Tabs>
+
+  </TabItem>
+</Tabs>
+
+### Advanced topics
+
+The following are some suggestions to improve the cluster setup:
```

Comment:
"Improve" word here would imply that there are issues with our guide. I would replace it with: The following are some advanced configuration topics to consider for your cluster.